ui/components/toolbar.go

```diff
@@ -13,14 +14,31 @@ func NewToolBar(
 	pullRequestMenuItemClickedEventHandler critic.PullRequestMenuItemClickedEventHandler,
 	analyzeButtonHandler critic.AnalyzeButtonClickedEventHandler,
```

Comment:
Is this unused now?

Reply:
they are both still used, passed into the toolbar actions as functions